config/_shared.json

```diff
@@ -1,3 +1,4 @@
 {
-	"discover_blog_id": 53424024
+	"discover_blog_id": 53424024,
+	"project": "project"
```

Comment:
Why don’t we use `calypso` as a project name? I assume you’d like to add another one called `mc`, right?